src/pics.c

```diff
@@ -607,11 +616,16 @@ int main_pics(int argc, char* argv[])
 			    || (ALGO_ADMM == algo)
 			    || (   (ALGO_NIHT == algo)
 				&& (regs[0].xform == NIHTWAV)));
+
+	// FIXME: will fail with looped dims
```

Comment:
can we then have an 'assert' or error message?

Reply:
added assert